### PR TITLE
Circles in links.html made to work

### PR DESCRIPTION
**Pull request by SilentFlame**
#97 resolved.
@niccokunzmann 
please have a review of this and provide feedback for further improvement.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Seems like it works. 
Looking at [Contributing.md](https://github.com/fossasia/flappy-svg/blob/master/Contributing.md). Please do one of these:
1. Give credit to the original creators of the picture according to their license
2. Assure that these pictures are your own work
3. Make these pictures your own work by rotoscoping them with Inkscape https://www.youtube.com/watch?v=dPHrmw4r16o and using different colors, ... You can use the SVG-images instead of png, then.

If we check in proprietary work into this repository, it could be that it will be blocked. We have a license CCby and if we have no right to put the work under CCby, it is against law.

**Comment by SilentFlame** [comment]
I have created this using Inkscape.
while adding this in place of the playstore image, do I still need some license to add?



If it's ok to add, then I'll design the other image too.

**Comment by niccokunzmann** [comment]
That is OK. You can design the other image, too.
A suggestion: Either make the black lines thick or remove them. They will not be visible at this size, so they should not be visible at all or always visible.

**Comment by SilentFlame** [comment]
@niccokunzmann  I didn't get what how to remove the last commits.

review the new images and guide me to add appropriate licenses.

**Comment by niccokunzmann** [comment]
The playstore logo looks good. I propose the following changes to it:
- [ ] Make it 256px*256px. File-> Document Properties -> Page and change the values
- [ ] You can add a license File-> Document Properties -> License -> CCby if you like
For the Android logo
- [ ] I still do not know where you got it from. Where did you get it from and what did the site say about usage of the logo?
- [ ] To create it with Inkscape you need
    - Circles
    - Rectangles
    - Text
    - Boolean operations, see [here](https://inkscape.org/en/doc/advanced/tutorial-advanced.html).

**Comment by SilentFlame** [comment]
I took the android logo from http://zenit.senecac.on.ca/wiki/index.php/APK 
where it is already under CCby License. so now what exactly we need to add here in this issue in the license.

I'm not getting what exactly you mean by File-> Document Properties -> License ,  File-> Document Properties -> Page.
where are these directories??

**Comment by niccokunzmann** [comment]
1. Can you help me find the CCby license on this page? It should be linked somewhere and we can take that text and put it into a  `apk.png.license` file. Also, we need a link to where we got it from in the "credits to" section of the credits.html.

2. When I open Inkscape, mine is German, so I can not check exctly the words, I can go like this: 
untitled
In license the second field is CCby and below you can add your name.

**Comment by niccokunzmann** [comment]
Okay, I found it! http://zenit.senecac.on.ca/wiki/index.php/Open_Source@Seneca:Copyrights

**Comment by SilentFlame** [comment]
How to add my name in the License I provided via Inkscape, as I don't see any slot there for writing my name.

**Comment by SilentFlame** [comment]
I'm going to add `apk.png.LICENSE` in the ```images``` directory only.
Is it ok? 

or do you want me to place it somewhere else?

**Comment by niccokunzmann** [comment]
The rightmost tab is only for the license. Have you clicked all tabs? I
have one where name and title and more can be written to.

Am 18.02.2016 um 12:32 schrieb Vinay Singh:

**Comment by niccokunzmann** [comment]
If you put `apk.png.LICENSE` next to `apk.png`, then it is the right place, where one would look for it.
I put the three Todos [here](https://github.com/fossasia/flappy-svg/pull/98#issuecomment-185612215).

I may be overdoing the credit giving a bit but I feel it is much better to give more credit than not enough. After all that person invested time so our life is easier. We should be thankful.

**Comment by SilentFlame** [comment]
@niccokunzmann apk.png.LICENSE and License to PlayStore image added and thickness of the lines also increased.
Unable to understand the working of squash, can you please guide me in that regard.

**Comment by SilentFlame** [comment]
sorry for closing and opening the pull request, that was by mistake.
hope it doesn't matter.

File: credits.html [hunk 1]
```diff
@@ -37,6 +37,7 @@
 					<ul>
 						<li><a href="http://codepen.io/elrumordelaluz/pen/XmygWX">Lionel T</a>(Github Edge)</li>
 						<li><a href="https://github.com/niccokunzmann/rotoscopes/">Rotoscopes</a></li>
+						<li><a href="http://zenit.senecac.on.ca/wiki/index.php/User:Dreyes6">Dan Reyes</a> for <a href="https://github.com/fossasia/flappy-svg/compare/images/apk.png?expand=1">apk.png</a></li>
```
Comment (niccokunzmann) [inline credits.html]: Please use the local path instead of "https://github.com/fossasia/flappy-svg/compare/images/apk.png?expand=1"

**Comment by niccokunzmann** [comment]
Cool! I feel a bad because I always find something to criticize. You fulfilled all other demands of mine.
Please consider this:
- [X] change the URL to the local one or to the site you got it from: "https://github.com/fossasia/flappy-svg/compare/images/apk.png?expand=1"
- [ ] resolve the merge conflicts.

It is a bit hard for me to explain how squash works... I just puts many commits in one commit and thus, if a file is added and deleted, it will not appear. I never used it. I will not demand that you use it.

**Comment by SilentFlame** [comment]
I don't see it as a criticize, It's a great guidance to learn and do things properly so I feel a helpful hand of your's always.

Changed the path, trying the squash thing.

**Comment by niccokunzmann** [comment]
Notify me, when you are done.

**Comment by SilentFlame** [comment]
@niccokunzmann  I think It's done now. Please have a look, and provide your feedback.

**Pull request merged**